src/config/auth.js

```diff
@@ -0,0 +1,7 @@
+export default {
+    facebook: {
```

Comment:
It is not safe to place clientSecret in the code,
It would be far for secure to get below values from environment variables